test-files/cmd/slow/Osm2OgrTranslation.sh.stdout

```diff
@@ -257,13 +257,13 @@ OGRFeature(foo_PAL015):0
  POINT : 
 
 #### Test to make sure all layers are read from a data source. ####
-19:51:37.020 WARN  ...?(  -1) CPL Error #: 1 Value '{"osm_id":"-162112","error:circular":"15","admin_leve":"4","hoot:layername":"planet_osm_line","boundary":"administrative","hoot:status":"invalid","z_order":"0","tags":"\"name:ar\"=>\"ماريلاند\", \"name:az\"=>\"Merilend\", \"name:bg\"=>\"Мериленд\", \"name:bn\"=>\"মেরিল্যান্ড\", \"name:el\"=>\"Μέριλαντ\", \"name:en\"=>\"Maryland\", \"name:eo\"=>\"Marilando\", \"name:fa\"=>\"مریلند\", \"name:fy\"=>\"Mary","name":"Maryland","ref":"MD"}' of field tags has been truncated to 254 characters.
-This warning will not be emitted any more for that layer.
-19:51:37.122 WARN  ...?(  -1) CPL Error #: 1 Value '{"place":"town","osm_id":"-2057485","error:circular":"15","admin_leve":"8","hoot:layername":"planet_osm_line","boundary":"administrative","hoot:status":"invalid","z_order":"0","tags":"\"is_in\"=>\"USA, Maryland\", \"tiger:CPI\"=>\"N\", \"wikipedia\"=>\"en:Delmar, Maryland\", \"tiger:LSAD\"=>\"43\", \"tiger:NAME\"=>\"Delmar\", \"border_type\"=>\"town\", \"is_in:state\"=>\"Maryland\", \"tiger:MTFCC\"=>\"G4110\", \"is_in:country\"=>\"USA\", \"tiger:CLASSFP\"=>\"C1\", \"tiger:PC","name":"Delmar"}' of field tags has been truncated to 254 characters.
-This warning will not be emitted any more for that layer.
-19:51:38.970 INFO  ...ore/io/OsmMapReaderFactory.cpp( 183) Loading map from test-output/cmd/slow/delaware-tds.shp...
+19:13:15.816 WARN  ...cpp/hoot/core/io/OgrWriter.cpp( 164) Truncateing the tags attribute (488 characters) to the output field width (254 characters).
+19:13:15.844 WARN  ...cpp/hoot/core/io/OgrWriter.cpp( 164) Truncateing the tags attribute (379 characters) to the output field width (254 characters).
+19:13:15.882 WARN  ...cpp/hoot/core/io/OgrWriter.cpp( 164) Truncateing the tags attribute (437 characters) to the output field width (254 characters).
```

Comment:
You are probably going to need to have an updated `test-files/cmd/slow/Osm2OgrTranslation.sh.stdout` file because of the warning output here.

Reply:
I thought I did. The change is from that file.  We are changing the warning from the driver to a hoot warning which means that we are not seeing the truncated tags. I'm not sure if we should put the truncated tags in the warning.